docs/version-4-upgrade.md

```diff
@@ -40,7 +41,7 @@ import * as firebase from 'firebase/app';
 
 user: Observable<firebase.User>;
 constructor(afAuth: AngularFireAuth) {
-  this.user = afAuth.authState;
+  this.user = afAuth.authState; // only triggered on sign-in/out (for old behavior use .idToken)
```

Comment:
Do we need more here to explain how to use both? It seems like _most_ users will just go with `.idToken` since it behaves like `authState` did. We should also update the main Authentication doc, but I guess we could do that after this PR is merged.